source/renderer/app/api/ada/getAdaHistoryByWallet.js

```diff
@@ -19,5 +23,5 @@ export const getAdaHistoryByWallet = (
     method: 'GET',
     path: '/api/v1/transactions',
     ...config,
-  }, requestParams)
+  }, requestParams, null, requestOptions)
```

Comment:
@daniloprates what does the null value represent here?

Reply:
It's the `rawBodyParams`